2.0/parser/filing_parser.py

```
@@ -0,0 +1,1799 @@
+            'business_address': None,
+            'business_phone': None,
+            'name_changes': [],
+            'header_raw_text': None,
+            'filing_raw_text': None
+        }
+        header_text = self.extract_sec_header(fulltext_contents)
+
+        if not header_text:
+            logging.error('Failed to extract SEC header from filing content.')
+            return filing_info
+        #filing_info['header_raw_text'] = header_text
```

Flake8 (7.2.0)

[error] block comment should start with '# '

(E265)